### DISCUSSION TITLE
Шрифт

### DISCUSSION THREAD

**Discussion by registant** [mindryder/ATLAS-RU#1] [ПРЕДЛОЖЕНИЕ ПЕРЕВОДА]
![Screenshot_1307](https://user-images.githubusercontent.com/123338303/213948497-bdcf9a1a-be88-47d6-b5a5-e9ededa1a5c6.jpg)
![Screenshot_1306](https://user-images.githubusercontent.com/123338303/213948499-0b3c2d41-8355-406e-a9a0-b58e80b55d09.jpg)

**Comment by registant** [comment]
Сам перевод -огромная работа. Спасибо за нее. Но вот шрифты на моих 1920 слишком мелкие и жирные. Они делают восприятие информации сложной и не подходят для PVP. Впрочем возможно это вопрос привычки.

**Comment by registant** [comment]
![Screenshot_1308](https://user-images.githubusercontent.com/123338303/213949108-bc098088-2ad9-489c-9667-0635ef06032a.jpg)

**Comment by registant** [comment]
При использовании вашего перевода и шрифта Roboto все становится более читабельным. Думаю еще поищу варианты.

**Comment by registant** [comment]
![Screenshot_1310](https://user-images.githubusercontent.com/123338303/213949571-82ae27d7-adb6-45c2-bec2-136f314b0083.jpg)

**Comment by mindryder** [comment]
Я большой противник прикручивания обычных шрифтов у которых нет стилизаций, я прекрасно понимаю что есть игроки привыкшие с такому (ведь откуда если с релиза они не видели нормального шрифта - лол, кроме американцев, и испанцев которые добавили позже), с моей субъективной точки зрения игра должна передавать больше атмосферы, и отсутствия шрифта в этом плане полностью разрушает атмосферность самой игры, превращая её в дрочильню. Основной шрифт и ведущий который максимально прилизан находится в папке русификатора и по мне вполне читабелен.
![AtlasGame_01xHMwpRje](https://user-images.githubusercontent.com/66848160/214048076-c3940797-7def-45df-9574-193b74732b4a.png)

Для альтернативы я добавил пару на выбор, возможно будут ещё, их надо компилировать на уровне движка, но все они будут хотя бы **МИНИМАЛЬНО СТИЛИЗОВАНЫ**. Ну и вас никто не ограничивает ставить свой обычный глобал и тому подобное :) 

Моя задача напомнить старым и новым игрокам, что это не АРК. И что эта игра с большим потенциалом с постройками кораблей и пиратским духом. А без шрифта с пиратским духом, это уже грустно и не красиво. 

**Так что обычные шрифты я никогда не выложу, даже в качестве альтернативы.**
![Atlas Screenshot 2023 01 22 - 00 55 41 83](https://user-images.githubusercontent.com/66848160/214049597-10bc865a-3d84-432d-adb7-1251e2ddb804.png)

**Comment by registant** [comment]
На самом деле на скрине, который я вижу - логи на русском. И мне как русскоговорящему человеку приятно и понятно читать их. В моей версии они на английском, и при наложении шрифта очень тяжелы для восприятия. Я не спорю с концепцией, мне нравится это ваше погружение. Выразил мнение только относительно чтения лога в режиме активного PVP. Буду тестировать новую версию.

**Reply by mindryder** [reply]
Я так и не понял пока почему на некоторых серверах логи на английском, когда они переведены.

**Reply by registant** [reply]
Забавно, я до последнего надеялся что это я что-то не так сделал. Ахахах. А проблема оказывается глубже. Ладно, с нетерпением жду! Логи на русском  это прям сильно.

**Reply by mindryder** [reply]
Логи на русском, переведены, но видимо на сервере на всех или нет не знаю, ставят принудительно английские

**Comment by registant** [comment]
![Screenshot_1371](https://user-images.githubusercontent.com/123338303/215027589-99ec9570-1c26-4f4b-bfe4-835696b603fd.jpg)
Обновил до 1.6 и снова вставил шрифт. Вот я о чем. Красиво, но максимально не читабельно(

**Reply by mindryder** [reply]
Неприлично много таймов (ну в этом случае помогает доп окно которое появляется), а с обычным шрифтом это как выглядит?

**Reply by Katranius** [reply]
> Обновил до 1.6 и снова вставил шрифт. Вот я о чем. Красиво, но максимально не читабельно(

Предложи шрифт, автор проверит как он в игре будет. Только это не два пальца об асфальт и 1-2 можно, но не более.

**Comment by registant** [comment]
![Screenshot_1386](https://user-images.githubusercontent.com/123338303/215255666-692db579-dd46-474d-a5fb-3793a4a62733.jpg)
